.buildkite/pipelines/security_solution/base.yml

```diff
@@ -0,0 +1,11 @@
+steps:
```

Comment:
How often to we plan to run it?

Reply:
On demand, when triggered by the quality gate, maybe in different PRs..... Not yet strictly defined